pkgs/by-name/fb/fbthrift/package.nix

```diff
@@ -100,22 +100,10 @@ stdenv.mkDerivation (finalAttrs: {
     ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       # Homebrew sets this, and the shared library build fails without
-      # it. I don‘t know, either. It scares me.
+      # it. I don’t know, either. It scares me.
```

Comment:
Out of curiosity, how did you notice this?

Reply:
I wrote this comment originally and the characters look a lot more distinct in my terminal font than the one used on GitHub. Also I’m an inveterate Unicode pedant in general :)